### DISCUSSION TITLE
Path traversal vulnerability on voila instance

### DISCUSSION THREAD

**Discussion by sustarun** [voila-dashboards/voila#1440] [Q&A]
Facing a security vulnerability (Path traversal) on a voila (v0.4) instance on a linux server. Would really appreciate any help or insight on this

**Comment by trungleduc** [comment]
Hi @sustarun, could you contact the person in [your previous issue](https://github.com/voila-dashboards/voila/issues/1438#issuecomment-1900345364) directly by email?
